### ISSUE TITLE
Dokieli + HTTP Sig

### ISSUE THREAD

**Issue by bblfish** [dokieli/dokieli#311]
[HTTP Sig](https://github.com/solid/authentication-panel/blob/main/proposals/HttpSignature.md) is a very efficient way to authenticate a client. It was initially implemented for [Reactive-Solid](https://github.com/co-operating-systems/Reactive-SoLiD) but the library is now being exported to the [httpSig repo](https://github.com/bblfish/httpSig) so that it can be generalised and used by Java, JS and node.JS environments. 

It is getting close to being testable in clients. So I am looking for Solid clients that could help guide the development of the library. Dokieli seems a good candidate :-)

**Comment by jeswr** [comment]
I was helping Henry try and get HTTP Sig support on CSS shortly before he passed (https://github.com/CommunitySolidServer/http-signature-component). Let me know if it would be useful for this to be picked back up when you get around to this issue and I'll see what I can do :)

**Comment by csarven** [comment]
Cool, happy to pick this up in dokieli - as you may already noticed above, it can be captured as part of the the [authentication mechanisms revisit milestone](https://github.com/dokieli/dokieli/milestone/6).

To be clear, both https://datatracker.ietf.org/doc/rfc9421/ and https://solid.github.io/httpsig/ are of interest for dokieli. As baseline, we need to get the former in place in dokieli, and a way for it to actually be usable alongside key management for instance. The latter needs more work in the context of Solid and can provide a way.

If we can move the implementation forward together, that'd be great. Would you be up for revisiting the HTTPSig in Solid spec with me as well?

**Comment by jeswr** [comment]
> Would you be up for revisiting the HTTPSig in Solid spec with me as well?

Yes - though I'll probably only have time to do so starting late November.

**Comment by VirginiaBalseiro** [comment]
> > Would you be up for revisiting the HTTPSig in Solid spec with me as well?
> 
> Yes - though I'll probably only have time to do so starting late November.

That works great with our timeline :) Thanks!
